<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Processing Collection Creator</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>313</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>118</y>
     <width>54</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Style sheet</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_css">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>114</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Select style sheet (CSS) file</string>
   </property>
   <property name="placeholderText">
    <string>Add image for icon</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_css">
   <property name="geometry">
    <rect>
     <x>387</x>
     <y>115</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>311</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_save">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>233</y>
     <width>76</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>About collection</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_icon">
   <property name="geometry">
    <rect>
     <x>387</x>
     <y>88</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_folder">
   <property name="geometry">
    <rect>
     <x>387</x>
     <y>10</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_folder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>10</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Folder</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>36</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_desc">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>36</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Long name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>62</y>
     <width>27</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>62</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Short name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_about">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>145</y>
     <width>321</width>
     <height>161</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>89</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Icon</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_icon">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>88</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Select icon file</string>
   </property>
   <property name="placeholderText">
    <string>Add image for icon</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
